peer.go

```diff
@@ -573,6 +580,7 @@ func (p *peer) addLink(chanPoint *wire.OutPoint,
 		UnsafeReplay:        cfg.UnsafeReplay,
 		MinFeeUpdateTimeout: htlcswitch.DefaultMinLinkFeeUpdateTimeout,
 		MaxFeeUpdateTimeout: htlcswitch.DefaultMaxLinkFeeUpdateTimeout,
+		ExpiryGraceDelta:    defaultBroadcastDelta + extraExpiryGraceDelta,
```

Comment:
How do we know that `defaultBroadcastDelta` is indeed the value that is used in `ChainArbitrator`? It currently is all the same value, but if the idea is that all configuration is done in `server.go`, shouldn't this value then also be passed in from `server.go`?

Reply:
My assumption was that since peer and server are both in the main package, server refers directly to defaults, and there's no existing peer config, that it'd be OK. But I agree that's it's probably better to pass it into `newPeer`. I'll make that change.